src/libs/OptionsListUtils.js

```diff
@@ -1941,13 +1941,23 @@ function shouldOptionShowTooltip(option) {
  * @param {Object} personalDetails
  * @param {Boolean} shouldGetOptionDetails
  * @param {Number} indexOffset
+ * @param {Boolean} maxOptionsSelected
  * @returns {Object}
  */
-function formatSectionsFromSearchTerm(searchTerm, selectedOptions, filteredRecentReports, filteredPersonalDetails, personalDetails = {}, shouldGetOptionDetails = false, indexOffset) {
-    // We show the selected participants at the top of the list when there is no search term
+function formatSectionsFromSearchTerm(
+    searchTerm,
+    selectedOptions,
+    filteredRecentReports,
+    filteredPersonalDetails,
+    personalDetails = {},
+    shouldGetOptionDetails = false,
+    indexOffset,
+    maxOptionsSelected,
```

Comment:
Hmm usually we don't want to add new params w/out a default **after** other params with a default (`personalDetails` and `shouldGetOptionDetails`) - but it looks like `indexOffset` was already added like that 😅 

Do y'all think we should swap the param order? I'd call this a NAB from me

Reply:
Yes, I think we should do this. There are only four instances. They can be carefully changed.